packages/modules/cra-template-web-viewer/template/src/App.tsx

```diff
@@ -41,7 +41,7 @@ const App: React.FC = () => {
       false
   );
   const [oidcInitialized, setOidcInitialized] = useState(false);
-  const [contextId, setContextId] = useState(process.env.IMJS_CONTEXT_ID);
+  const [iTwinId, setContextId] = useState(process.env.IMJS_CONTEXT_ID);
```

Comment:
should the env var be renamed too? i think so but just wanted to confirm

Reply:
yes. Might as well